### DISCUSSION TITLE
bypy安装在armbian上，用ssh访问下载文件，连接断掉之后，之前的下载还在继续吗，如何查看进度和状态？

### DISCUSSION THREAD

**Discussion by tjuarch** [houtianze/bypy#669] [General]
bypy安装在机顶盒的armbian系统里，只能远程ssh用root访问，远程下载文件都正常，但是断开ssh连接后，再登录不知道bypy的状态是什么样的，是否在后台下载，或者怎么查看，请教：）

**Comment by houtianze** [comment]
这个你搜索一下`nohup`或者`tmux`相关信息，就是如何实现logout进程能继续跑的问题。

**Comment by nell332** [comment]
这个系统有screen命令吗，有的话用screen bypy就可以，断开ssh后想恢复直接用screen -r

**Comment by tjuarch** [comment]
谢谢，按您说的找到了方法发自我的 iPhone在 2024年1月16日，17:28，nell332 ***@***.***> 写道：﻿
这个系统有screen命令吗，有的话用screen bypy就可以，断开ssh后想恢复直接用screen -r
